lib/prawn/vera_pdf.rb

```diff
@@ -0,0 +1,55 @@
+require 'nokogiri'
+require 'open3'
+
+module Prawn
+  module VeraPdf
```

Comment:
I am more used to GitLab which lets you 'resolve' a discussion manually. I use my 'dones' here primarily for keeping track of what I still need to do. For example your comment about making sure that the CI runs the veraPDF specs is now hidden as 'outdated' because I moved the file elsewhere. As long as you are not annoyed with getting emails for every 'done' I would continue with this practice. Or is there some GitHub trick for such things? :grin: